src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/listeners/HMPPSTierListener.kt

```diff
@@ -38,19 +39,23 @@ class HMPPSTierListener(
           telemetryService.successfulWrite(crn, calculationId)
         }
       }
+    } catch (e: WebClientResponseException.NotFound) {
+      if (e.responseBodyAsString.contains("Offender with CRN 12345 not found")) {
+        telemetryService.offenderNotFoundFailedWrite(crn, calculationId)
```

Comment:
Not throwing an error here will return success and the message will not go back onto the queue